eventual-cassandra/src/test/scala/com/evolutiongaming/kafka/journal/eventual/cassandra/ReplicatedCassandraTest.scala

```diff
@@ -890,7 +891,9 @@ class ReplicatedCassandraTest extends AnyFunSuite with Matchers {
           Offset.unsafe(1),
           timestamp   = timestamp0,
           expireAfter = none,
-          events = Nel.of(eventRecordOf(seqNr = SeqNr.min, partitionOffset = PartitionOffset(Partition.min, Offset.min)).event),
+          events = Nel.of(
+            eventRecordOf(seqNr = SeqNr.min, partitionOffset = PartitionOffset(Partition.min, Offset.unsafe(1))).event,
```

Comment:
Please elaborate why `Offset.min` was changed on `Offset.unsafe(1)`

Reply:
to make it obvious what value is used and to make it consistent with other lines where `Offset.unsafe(<other value>)` gets used. It doesn't carry any meaning in this test case.